packages/design-system-docs/src/pages/components/Dropdown/Dropdown.example.jsx

```diff
@@ -12,12 +12,23 @@ const dropdownOptions = [
   { label: 'Option 7', value: '7' },
   { label: 'Option 8', value: '8' },
 ];
+const dropdownOptions2 = [
+  { label: '--Please select an option--', value: '' },
```

Comment:
Tiny thing, but maybe the default could be:
```suggestion
  { label: '- Select an option -', value: '' },
```